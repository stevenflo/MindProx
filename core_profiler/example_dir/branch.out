# Mix output version 2
# FINI: end of program
# EMIT_DYNAMIC_STATS: instructions = 99232
Branch Stats: Cond Branches = 17217 Uncond branches = 2210 Tk branches = 9095
Branch Bucket 0 is equal to 0
Branch Bucket 1 is equal to 2845
Branch Bucket 2 is equal to 6250
Branch Transition Bucket 0 is equal to 12271
Branch Transition Bucket 1 is equal to 3126
Branch Transition Bucket 2 is equal to 596
Branch Transition Bucket 3 is equal to 247
Branch Transition Bucket 4 is equal to 116
Branch Transition Bucket 5 is equal to 225
Branch Transition Bucket 6 is equal to 225
Branch Transition Bucket 7 is equal to 26
Branch Transition Bucket 8 is equal to 6
Branch Transition Bucket 9 is equal to 379
# END_DYNAMIC_STATS
