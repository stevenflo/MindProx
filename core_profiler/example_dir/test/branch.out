# Mix output version 2
# FINI: end of program
# EMIT_DYNAMIC_STATS: instructions = 96442
Branch Stats: Cond Branches = 16623 Uncond branches = 2084 Tk branches = 8569
Branch Bucket 0 is equal to 0
Branch Bucket 1 is equal to 2648
Branch Bucket 2 is equal to 5921
Branch Transition Bucket 0 is equal to 11903
Branch Transition Bucket 1 is equal to 3141
Branch Transition Bucket 2 is equal to 574
Branch Transition Bucket 3 is equal to 224
Branch Transition Bucket 4 is equal to 82
Branch Transition Bucket 5 is equal to 174
Branch Transition Bucket 6 is equal to 242
Branch Transition Bucket 7 is equal to 30
Branch Transition Bucket 8 is equal to 6
Branch Transition Bucket 9 is equal to 247
# END_DYNAMIC_STATS
